src/test/regress/sql/multi_modifying_xacts.sql

```diff
@@ -1064,7 +1068,20 @@ SET search_path TO multi_modifying_xacts;
 SET citus.next_shard_id TO 1200020;
 SET citus.next_placement_id TO 1200033;
```

Comment:
These lines should be after your newly added lines. Because now the shards created later on don't have consistent shard ids.

Reply:
Added